implementations/rust/ockam/ockam_transport_tcp/src/workers/sender.rs

```diff
@@ -251,3 +256,17 @@ fn prepare_message(msg: TransportMessage) -> Result<Vec<u8>> {
 
     Ok(msg_buf)
 }
+
+#[cfg(test)]
+mod test {
+    use ockam_core::route;
+
+    use super::*;
+
+    #[test]
+    fn prepare_message_should_discard_large_messages() {
+        let msg = TransportMessage::v1(route![], route![], vec![0; u16::MAX as usize + 1]);
```

Comment:
Perhaps we should use the `MAXIMUM_MESSAGE_LENGTH` constant here instead of the `u16:MAX`.

Reply:
the counterargument would be that if someone (inadvertently) changes MAXIMUM_MESSAGE_LENGTH, then this test may fail (in case it's increased), so it would act as a guardrail for that situation.